community/releases/release-manager-runbook/index.md

```diff
@@ -208,8 +221,9 @@ before merging.
     1. **Spinnaker Release Alias** should be the name of a Netflix original TV
     show converted to an alphanumeric string
     (e.g., "Gilmore Girls A Year in the Life").
+    Only worry about name collisions with the current active releases (i.e. `hal version list`)
 
-    1. **BOM version** should be "release-1.nn.x-latest-validated" (replacing nn
+    1. **BOM version** should be `release-1.nn.x-latest-unvalidated` (replacing nn
```

Comment:
s/should/must/
or
s/should/will/
or
s/should/is/

Whichever makes the most sense to you. But saying "should" implies it's optional.